controllers/salida.go

```diff
@@ -74,17 +74,19 @@ func (c *SalidaController) Post() {
 			if respuesta, err := salidaHelper.PostTrSalidas(&v); err == nil && respuesta != nil {
 				c.Ctx.Output.SetStatus(201)
 				c.Data["json"] = respuesta
-			} else if err != nil {
-				panic(map[string]interface{}{
-					"funcion": "Post - salidaHelper.PostTrSalidas(&v)",
-					"err":     err,
-					"status":  "400",
-				})
 			} else {
+				status := "400"
+				if err == nil {
+					err = map[string]interface{}{
+						"err": errors.New("No se obtuvo respuesta al registrar la(s) salida(s)"),
+					}
```

Comment:
Dado que mas adelante el error se está encapsulando, no es necesario encapsularlo nuevamente/desde-antes
```suggestion
					err = errors.New("No se obtuvo respuesta al registrar la(s) salida(s)")
```

Reply:
Con el cambio sugerido falla el build porque en este contexto la variable err es de tipo map[string]interface{} y no Error, por lo que se debió encapsular de esa manera.